### DISCUSSION TITLE
Will SD-JWT be used to meet the Verifiable Credentials Data Model v1.1 requirement?

### DISCUSSION THREAD

**Discussion by sander** [eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework#357] [Q&A]
CIR (EU) 2024/2979 Art. 8 requires wallet solutions to support ISO/IEC 18013-5:2021 and W3C VCDM v1.1.

The ARF refers to SD-JWT VC in ongoing work including #356. In [draft-ietf-oauth-sd-jwt-vc-08](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-sd-jwt-vc-08) § 1.2 the reader is warned:

> Note: This specification does not utilize the W3C's Verifiable Credentials Data Model v1.0, v1.1, or v2.0.

There is ongoing work in [sd-jwt-vc-dm](https://github.com/danielfett/sd-jwt-vc-dm) by @danielfett to update the Internet-Draft for W3C VCDM v2.0 compatibility.

Is the interpretation that SD-JWT VC implementations can eventually comply to W3C VCDM v1.1? Is it because of this ongoing work, or has the necessary work already been done? Is W3C VCDM v2.0 sufficiently backwards-compatible with v1.1 for this purpose? Is the Commission also considering other digital document formats implementing W3C VCDM v1.1?

**Comment by danielfett** [comment]
Hi Sander,

the current version of SD-JWT VC is meant to be compatible to VCDM in the sense that existing W3C VCDM data structures can be re-used on top of SD-JWT VC. The sentence you quoted is to say that the draft itself doesn't depend on anything defined in W3C but does not intend to preclude compatibility.

That said, there has been a lot of confusion around this topic, partly due to how we positioned SD-JWT VCDM, the sentence you quoted and other wordings and things not being defined precise enough. We have recently started an activity to clear that up, with modifications to SD-JWT VC and a potentially more normative status of SD-JWT VCDM (right now, it is just a document floating around on my github). Part of this activity will also be to go through all potential incompatibilities between the formats and define how they should be treated.

**Reply by sander** [reply]
Thank you, this clarifies to me that SD-JWT VC implementations can ~~eventually~~ <ins>already</ins> conform to some version of W3C VCDM. Good to hear about the work to clear it up.

Is the intention that they can conform to W3C VCDM v1.1 as well as v2.0?

Do you know if there are other formats than SD-JWT VC being considered for W3C VCDM v1.1 conformance?
